commit 20b5d91422f436aff0e7f73fccecbd335cc0af6e (refs/changes/64/1064/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-12-05T23:41:33+00:00 (2 years, 10 months ago)
    
    IMGAPI-601 IMGAPI should allow '+' in a manifest version
    Reviewed by: Todd Whiteman <todd.whiteman@joyent.com>

diff --git a/docs/index.md b/docs/index.md
index e34980e..20a969c 100644
--- a/docs/index.md
+++ b/docs/index.md
@@ -4,15 +4,6 @@ markdown2extras: tables, code-friendly, cuddled-lists, link-patterns
 markdown2linkpatternsfile: link-patterns.txt
 apisections: Images, Channels, Miscellaneous API
 ---
-<!--
-    This Source Code Form is subject to the terms of the Mozilla Public
-    License, v. 2.0. If a copy of the MPL was not distributed with this
-    file, You can obtain one at http://mozilla.org/MPL/2.0/.
--->
-
-<!--
-    Copyright (c) 2016, Joyent, Inc.
--->
 
 # Image API (IMGAPI)
 
@@ -201,13 +192,22 @@ A version string for this image. Maximum 128 characters. This is an opaque
 string, i.e. no particular format or structure is enforced and
 no ordering with other versions is implied. However, it is strongly suggested
 that the [semver](http://semver.org/) versioning scheme be
-followed. Further, the simple `Major.Minor.Patch` semver subset is ideal.
+followed.
 
 Note that image `name` and `version` do not make a unique identifier for
 an image. Separate users (and even the same user) can create images with
 the same name and version. The image `uuid` is the only unique identifier
 for an image.
 
+Starting in IMGAPI v3.2.0, support was added to allow '+' in the "version"
+field, because this is one of the characters [allowed by
+semver](http://semver.org/#spec-item-10). However, it wasn't until OS-5798 that
+`imgadm` (v3.7.0) in the platform was update to allow '+' in a version.
+Therefore a **warning**: do not use '+' in an image version until you know that
+the minimum platform version for any server in your target DC(s) is greater than
+or equal to 20161118T231131Z (when OS-5798 was
+[integrated](https://github.com/joyent/smartos-live/commit/fc5816a)).
+
 
 ## Manifest: description
 
diff --git a/docs/link-patterns.txt b/docs/link-patterns.txt
index 2e2a694..a3f4261 100644
--- a/docs/link-patterns.txt
+++ b/docs/link-patterns.txt
@@ -1 +1 @@
-/([A-Z]+-\d+)/ https://devhub.joyent.com/jira/browse/\1
+/(?<!/)(\b[A-Z]+-\d+\b)/ https://smartos.org/bugview/\1
diff --git a/package.json b/package.json
index 69c551a..9fe57a7 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "imgapi",
   "description": "Image API to manage images for SDC 7",
-  "version": "3.1.3",
+  "version": "3.2.0",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
@@ -17,7 +17,7 @@
     "forkexec": "1.1.0",
     "glob": "7.0.5",
     "handlebars": "4.0.5",
-    "imgmanifest": "git+https://github.com/joyent/node-imgmanifest.git#91e5d80",
+    "imgmanifest": "2.3.0",
     "json": "9.0.4",
     "ldap-filter": "0.3.1",
     "uuid": "2.0.2",
